debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -149,11 +162,45 @@ public void configure(Configuration config, HistoryRecordComparator comparator)
     @Override
     public synchronized void start() {
         super.start();
+        createDatabaseHistoryTopic();
```

Comment:
Instead of doing this here, I think a better place to call `createDatabaseHistoryTopic()` would be to trigger this from `MySqlConnectorTask#start()`, if the topic not exists. This seems better in terms of overall control flow also avoids the issue of unneeded admin client instantiation pointed out by @pgoranss. The routine below gets a bit simpler then, too, as we don't need to check for existance of the topic.